src/pages/plans/splitPlans/UpperBody.jsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
```

Comment:
Delete this file, it should not be part of this PR (and it cannot be displayed through menu click).